lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +93,39 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
+            config = json.load(file)
+        return ConfigDTO(**config)
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters are not corrupt.
         """
+        if not isinstance(self.config.seed_urls, list):
+            raise IncorrectSeedURLError("Seed URL does not match standard pattern 'https?://(www.)?'")
```

Comment:
error message (not type) does not reflect actual problem that you identified

Reply:
changed it